flink-kubernetes/src/test/java/org/apache/flink/kubernetes/KubernetesClientTestBase.java

```diff
@@ -183,6 +183,29 @@ protected void mockPodEventWithLabels(
                 .once();
     }
 
+    protected void mockPodEventWithLabelsFail(
```

Comment:
This name does not reflect what this method really does. I'd suggest `mockWatchPodFailure`